google/cloud/bigtable_admin/__init__.py

```diff
@@ -13,439 +13,12 @@
-    "ListInstancesRequest",
-    "ListInstancesResponse",
-    "ListLogicalViewsRequest",
-    "ListLogicalViewsResponse",
-    "ListMaterializedViewsRequest",
-    "ListMaterializedViewsResponse",
-    "PartialUpdateClusterMetadata",
-    "PartialUpdateClusterRequest",
-    "PartialUpdateInstanceRequest",
-    "UpdateAppProfileMetadata",
-    "UpdateAppProfileRequest",
-    "UpdateClusterMetadata",
-    "UpdateInstanceMetadata",
-    "UpdateLogicalViewMetadata",
-    "UpdateLogicalViewRequest",
-    "UpdateMaterializedViewMetadata",
-    "UpdateMaterializedViewRequest",
-    "CheckConsistencyRequest",
-    "CheckConsistencyResponse",
-    "CopyBackupMetadata",
-    "CopyBackupRequest",
-    "CreateAuthorizedViewMetadata",
-    "CreateAuthorizedViewRequest",
-    "CreateBackupMetadata",
-    "CreateBackupRequest",
-    "CreateSchemaBundleMetadata",
-    "CreateSchemaBundleRequest",
-    "CreateTableFromSnapshotMetadata",
-    "CreateTableFromSnapshotRequest",
-    "CreateTableRequest",
-    "DataBoostReadLocalWrites",
-    "DeleteAuthorizedViewRequest",
-    "DeleteBackupRequest",
-    "DeleteSchemaBundleRequest",
-    "DeleteSnapshotRequest",
-    "DeleteTableRequest",
-    "DropRowRangeRequest",
-    "GenerateConsistencyTokenRequest",
-    "GenerateConsistencyTokenResponse",
-    "GetAuthorizedViewRequest",
-    "GetBackupRequest",
-    "GetSchemaBundleRequest",
-    "GetSnapshotRequest",
-    "GetTableRequest",
-    "ListAuthorizedViewsRequest",
-    "ListAuthorizedViewsResponse",
-    "ListBackupsRequest",
-    "ListBackupsResponse",
-    "ListSchemaBundlesRequest",
-    "ListSchemaBundlesResponse",
-    "ListSnapshotsRequest",
-    "ListSnapshotsResponse",
-    "ListTablesRequest",
-    "ListTablesResponse",
-    "ModifyColumnFamiliesRequest",
-    "OptimizeRestoredTableMetadata",
-    "RestoreTableMetadata",
-    "RestoreTableRequest",
-    "SnapshotTableMetadata",
-    "SnapshotTableRequest",
-    "StandardReadRemoteWrites",
-    "UndeleteTableMetadata",
-    "UndeleteTableRequest",
-    "UpdateAuthorizedViewMetadata",
-    "UpdateAuthorizedViewRequest",
-    "UpdateBackupRequest",
-    "UpdateSchemaBundleMetadata",
-    "UpdateSchemaBundleRequest",
-    "UpdateTableMetadata",
-    "UpdateTableRequest",
-    "OperationProgress",
-    "StorageType",
-    "AppProfile",
-    "AutoscalingLimits",
-    "AutoscalingTargets",
-    "Cluster",
-    "HotTablet",
-    "Instance",
-    "LogicalView",
-    "MaterializedView",
-    "AuthorizedView",
-    "Backup",
-    "BackupInfo",
-    "ChangeStreamConfig",
-    "ColumnFamily",
-    "EncryptionInfo",
-    "GcRule",
-    "ProtoSchema",
-    "RestoreInfo",
-    "SchemaBundle",
-    "Snapshot",
-    "Table",
-    "RestoreSourceType",
-    "Type",
-)
-
-import google.cloud.bigtable_admin_v2.overlay  # noqa: F401
-from google.cloud.bigtable_admin_v2.overlay import *  # noqa: F401, F403
-
-__all__ += google.cloud.bigtable_admin_v2.overlay.__all__
+__all__ = google.cloud.bigtable.admin.__all__
```

Comment:
Do we have test that try importing in different ways, to make sure they all work the same as before?

i.e. `import google.cloud.bigtable.admin.Type`, `from google.cloud.bigtable.admin import Type`, etc